dragonchain/broadcast_processor/broadcast_functions.py

```diff
@@ -176,6 +196,27 @@ def schedule_block_for_broadcast_sync(block_id: str, time: int = 0) -> None:
     await redis.zadd_async(IN_FLIGHT_KEY, time, block_id)
 
 
+def schedule_notification_for_broadcast_sync(notification_location: str, time: int = 0) -> None:
+    """Schedule a certain notification to be checked by the broadcast processor (sync)
+    Args:
+        notification_location: notification_location to schedule
+        time: unix timestamp to re-schedule for checking (0 means ASAP)
+    """
+    redis.lpush_sync(NOTIFICATION_KEY, notification_location)
+
+
+def verification_storage_location(l1_block_id: str, level_received_from, chain_id) -> str:
```

Comment:
prefer type annotations in the actual signature rather than the docstring

Reply:
agree